src/viur/core/bones/relational.py

```diff
@@ -405,74 +405,53 @@ def serialize(self, skel: "SkeletonInstance", name: str, parentIndexed: bool) ->
 
         :raises AssertionError: If a programming error is detected.
         """
+
+        def serialize_dest_rel(in_value: dict | None = None):
+            if not in_value:
+                return None, None
+            if dest_val := in_value.get("dest"):
+                ref_data_serialized = dest_val.serialize(parentIndexed=indexed)
+            else:
+                ref_data_serialized = None
```

Comment:
I wonder if this case must be checked so far? IMHO, a RelationalBone's value is invalid when there is no "dest" provided.